packages/circle-demo-webapp/app/components/ui/avatar.tsx

```diff
@@ -0,0 +1,62 @@
+import * as AvatarPrimitive from '@radix-ui/react-avatar';
+import * as React from 'react';
+
+import { cn } from '~/lib/utils';
+
+type AvatarProps = React.ComponentPropsWithoutRef<typeof AvatarPrimitive.Root> & {
+  className?: string;
+};
+
+const Avatar = React.forwardRef<
+  React.ElementRef<typeof AvatarPrimitive.Root>,
```

Comment:
all components inside /ui folder are imported to the project with shadcn CLI tool: https://ui.shadcn.com/docs/cli
if they are not used right now, they will be used soon